pkg/reconciler/route/route.go

```diff
@@ -302,13 +313,32 @@ func (c *Reconciler) reconcile(ctx context.Context, r *v1alpha1.Route) error {
 		Hostname: resourcenames.K8sServiceFullname(r),
 	}
 
+	tls := []netv1alpha1.ClusterIngressTLS{}
+	if config.FromContext(ctx).Network.AutoTLS {
+		domains := tr.GetDomains(r.Status.URL.Host, traffic.Targets)
+		desiredCert := resources.MakeCertificate(r, domains)
+		cert, err := c.reconcileCertificate(ctx, r, desiredCert)
+		if err != nil {
+			r.Status.MarkCertificateProvisionFailed(desiredCert.Name)
+			return err
+		}
+
+		if cert.Status.IsReady() {
+			r.Status.MarkCertificateReady(cert.Name)
+		} else {
+			r.Status.MarkCertificateNotReady(cert.Name)
```

Comment:
Here we should probably make the URLs use `http`

Reply:
Done